modules/microshift-get-rpm-release-info.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+//microshift_updating/microshift-list-update-contents.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-get-rpm-release-info_{context}"]
+= List the contents of the {microshift-short} RPM release package
+
+To see the images included with a {microshift-short} release, you can list the contents of the `microshift-release-info` RPM by downloading and unpacking the RPM.
+
+.Prerequisites
+
+* You are using {op-system-base-full} or an operating system with an RPM package manager.
+* Your {microshift-short} repositories are enabled.
+
+.Procedure
+
+. Optional: List which versions of the {microshift-short} RPM are available for download by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf repoquery microshift-release-info-0:4.16.* # <1>
+----
+<1> Replace the example value `4.16.*` with the major and minor release numbers you are interested in.
++
+.Example output
+[source,terminal]
+----
+Updating Subscription Management repositories.
+microshift-release-info-0:4.16.0-202406260523.p0.gc5a37df.assembly.4.16.0.el9.noarch
+microshift-release-info-0:4.16.1-202406281132.p0.g8babeb9.assembly.4.16.1.el9.noarch
+microshift-release-info-0:4.16.10-202408291007.p0.g6e4ee4d.assembly.4.16.10.el9.noarch
+microshift-release-info-0:4.16.2-202407040825.p0.g2e0407e.assembly.4.16.2.el9.noarch
+microshift-release-info-0:4.16.3-202407111123.p0.ge4206d3.assembly.4.16.3.el9.noarch
+microshift-release-info-0:4.16.4-202407191908.p0.g057a9af.assembly.4.16.4.el9.noarch
+microshift-release-info-0:4.16.5-202407250951.p0.g0afcb57.assembly.4.16.5.el9.noarch
+microshift-release-info-0:4.16.6-202408010822.p0.gc4ded66.assembly.4.16.6.el9.noarch
+microshift-release-info-0:4.16.7-202408081107.p0.g0597bb8.assembly.4.16.7.el9.noarch
+microshift-release-info-0:4.16.8-202408150851.p0.gc8a3bb1.assembly.4.16.8.el9.noarch
+microshift-release-info-0:4.16.9-202408220842.p0.gefa92a2.assembly.4.16.9.el9.noarch
+----
+
+. Download the RPM package you want by running the following command:
++
+--
+[source,terminal]
+----
+$ sudo dnf download microshift-release-info-_<release_version>_ # <1>
+----
+<1> Replace `_<release_version>_` with the numerical value of the release you are deploying, using the entire version number, for example, `4.16.0`.
+--
++
+--
+.Example output
+[source,terminal]
+----
+microshift-release-info-4.16.0.*.el9.noarch.rpm # <1>
+----
+<1> The `*` represents the date and commit ID. Your output should contain both, for example, `-202311101230.p0.g7dc6a00.assembly.4.16.0`.
+--
+
+. Unpack the RPM package without installing it by running the following command:
++
+[source,terminal]
+----
+$ rpm2cpio _<my_microshift_release_info>_ | cpio -idmv # <1>
+./usr/share/microshift/release/release-aarch64.json
```

Comment:
Should these two lines be removed? They look like example output (which should be separate from commands), but it looks like they're already included in the example output block below